docs/source/quicktour.mdx

```diff
@@ -14,186 +14,186 @@ specific language governing permissions and limitations under the License.
 
 ```bash
 pip install torch
 ===PT-TF-SPLIT===
 pip install tensorflow
 ```
 
-```py
+Import the [`pipeline`] with the task you want to complete:
+
+```python
 >>> from transformers import pipeline
->>> classifier = pipeline('sentiment-analysis')
+>>> classifier = pipeline("sentiment-analysis")
 ```
 
-When typing this command for the first time, a pretrained model and its tokenizer are downloaded and cached. We will
-look at both later on, but as an introduction the tokenizer's job is to preprocess the text for the model, which is
-then responsible for making predictions. The pipeline groups all of that together, and post-process the predictions to
-make them readable. For instance:
+The pipeline downloads and caches a default [pretrained model](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) and tokenizer for sentiment analysis. Now you can use the `classifier` on your target text:
 
-
-```py
->>> classifier('We are very happy to show you the 🤗 Transformers library.')
-[{'label': 'POSITIVE', 'score': 0.9998}]
+```python
+>>> classifier("We are very happy to show you the 🤗 Transformers library."")
+[{"label": "POSITIVE", "score": 0.9998}]
 ```
 
-That's encouraging! You can use it on a list of sentences, which will be preprocessed then fed to the model, returning
-a list of dictionaries like this one:
+For more than one sentence, pass a list of sentences to the [`pipeline`] which returns a list of dictionaries:
 
-```py
+```python
 >>> results = classifier(["We are very happy to show you the 🤗 Transformers library.",
 ...            "We hope you don't hate it."])
 >>> for result in results:
-...     print(f"label: {result['label']}, with score: {round(result['score'], 4)}")
+...     print(f"label: {result["label"]}, with score: {round(result["score"], 4)}")
 label: POSITIVE, with score: 0.9998
 label: NEGATIVE, with score: 0.5309
 ```
 
-To use with a large dataset, look at [iterating over a pipeline](main_classes/pipelines)
+The [`pipeline`] can also iterate over an entire dataset. Start by installing the 🤗 Datasets library:
+
+```bash
+pip install datasets 
+```
+
+Create a [`pipeline`] with the task you want to solve for, and the model you want to use:
 
-You can see the second sentence has been classified as negative (it needs to be positive or negative) but its score is
-fairly neutral.
+```python
+>>> from transformers import pipeline
+>>> pipe = pipeline("automatic-speech-recognition", model="facebook/wav2vec2-base-960h", device=0)
+```
 
-By default, the model downloaded for this pipeline is called "distilbert-base-uncased-finetuned-sst-2-english". We can
-look at its [model page](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) to get more
-information about it. It uses the [DistilBERT architecture](model_doc/distilbert) and has been fine-tuned on a
-dataset called SST-2 for the sentiment analysis task.
+Load the dataset you want to iterate over:
```

Comment:
The documentation to `load_dataset` or the quicktour of Datasets should be linked here.